conf/application.conf

```diff
@@ -183,6 +183,11 @@ microservice {
       port = 8650
     }
 
+    citizen-details {
```

Comment:
Ensure we override this in `app-config-base`

Reply:
PR : https://github.com/hmrc/app-config-base/pull/5943